test/TestLogger.AcceptanceTests/Snapshots/TestLoggerAcceptanceTests/VerifyTestRunOutput/Json.TestLogger.NUnit.NetFull.Tests-WindowsOnly.verified.txt

```diff
@@ -318,67 +238,51 @@ ExampleTest4(System.AggregateException: One or more errors occurred. ---> NUnit.
         Name: ParametrizedTestCases,
         Tests: [
           {
-            FullyQualifiedName: NUnit.Xml.TestLogger.NetFull.Tests.ParametrizedTestCases.TestData(1,"A"),
-            DisplayName: TestData(1,"A"),
             Namespace: NUnit.Xml.TestLogger.NetFull.Tests,
             Type: ParametrizedTestCases,
-            Method: TestData(1,"A"),
+            Method: TestData(1,\"A\"),
             Result: Passed
           },
           {
-            FullyQualifiedName: NUnit.Xml.TestLogger.NetFull.Tests.ParametrizedTestCases.TestData(1,"B"),
-            DisplayName: TestData(1,"B"),
             Namespace: NUnit.Xml.TestLogger.NetFull.Tests,
             Type: ParametrizedTestCases,
-            Method: TestData(1,"B"),
+            Method: TestData(1,\"B\"),
             Result: Passed
           },
           {
-            FullyQualifiedName: NUnit.Xml.TestLogger.NetFull.Tests.ParametrizedTestCases.TestData(2,"A"),
-            DisplayName: TestData(2,"A"),
             Namespace: NUnit.Xml.TestLogger.NetFull.Tests,
             Type: ParametrizedTestCases,
-            Method: TestData(2,"A"),
+            Method: TestData(2,\"A\"),
```

Comment:
I don't think we have another thread about this. We get this kind of change from the sanitizer running. They look a bit nonsensical because Verifier is using non strict Json in its output. Strict json would look like `Method: "TestData(2,\"A\")",` so the escaping makes more sense.

Reply:
Yes, this is alright for JSON serializer. I hope we're not escaping for XML serializer.

Reply:
No, we aren't.